ctgan/synthesizers/ctgan.py

```diff
@@ -405,7 +405,7 @@ def fit(self, train_data, discrete_columns=tuple(), epochs=None):
                 optimizerG.step()
 
             if self._verbose:
-                print(f"Epoch {i+1}, Loss G: {loss_g.detach().cpu(): .4f},"
+                print(f"Epoch {i+1}, Loss G: {loss_g.detach().cpu(): .4f},"  # noqa: T001
```

Comment:
I have two concerns here:
1. `"` quotes, change them to `'` otherwise `flake8-quotes` will fail after merge.
2. Do we want to keep the prints or use `LOGGER` instead ?

Reply:
I think `prints` should be kept in this case, since all the previous users are already familiar with these outputs.
In the future we may want to further extend the `verbose` option to enable tqdm instead of prints (e.g. `verbose='tqdm'`), but even then these prints should be kept if `verbose=True`.